scripts/Muon_Analysis_2.py

```diff
@@ -0,0 +1,52 @@
+# pylint: disable=invalid-name
+from __future__ import (absolute_import, division, print_function)
+
+import sys
+
+import PyQt4.QtGui as QtGui
+import PyQt4.QtCore as QtCore
+
+from Muon import dummy_label_widget
+from Muon import dock_widget
+
+
+class MuonAnalysis2Gui(QtGui.QMainWindow):
+
+    def __init__(self, parent=None):
+        super(MuonAnalysis2Gui, self).__init__(parent)
+
+        loadWidget = dummy_label_widget.DummyLabelWidget("Load dummy", self)
```

Comment:
Can you help me understand why we wouldn't use the Qt designer for a lot of the layout work?

Reply:
Personally I prefer doing it through the code, as I find it easier to understand. It also keeps everything in one place. I feel that it shouldn't make too much difference as we are primarily just adding widgets

Reply:
Ok. I would say that one thing to consider if things get much more complicated is that maintenance in the long term can be easier with a `.ui` files as changing the layout is easier with just dragging things around.